packages/desktop/src/renderer/components/widgets/channels/BasicMessage.tsx

```diff
@@ -207,6 +210,11 @@ export const BasicMessageComponent: React.FC<BasicMessageProps & FileActionsProp
                       {infoMessage ? 'Quiet' : messageDisplayData.nickname}
                     </Typography>
                   </Grid>
+                  {userLabel && (
+                    <Grid>
+                      <UserLabelContainer type={userLabel} username={messageDisplayData.nickname} />
```

Comment:
Why did you add "container" suffix?

Reply:
Ok, I see. Won't it break storybook/snapshot testing? I guess there should probably not be a separate container for user label only.